### ISSUE TITLE
macos 0.6.0 version/help/usage problems

### ISSUE THREAD

**Issue by silverdr** [MitMaro/git-interactive-rebase-tool#58]
After downloading [0.6.0 version for macos](https://github.com/MitMaro/git-interactive-rebase-tool/releases/download/0.6.0/macos-git-interactive-rebase-tool) I tried:
```
$ interactive-rebase-tool --help
Error opening file, --help
```
```
$ interactive-rebase-tool
Must provide a filepath.

Usage: git-interactive <filepath>
```
```
$ interactive-rebase-tool --version
v0.5.0
```
Which means that basically no single invocation returned expected response. First didn't recognise the `--help` switch even if the switch is listed in `README.md`. Second shows incorrect tool name (`git-interactive` instead of `interactive-rebase-tool` as it is supposed to be called, according to `README.md`. Eventually the last one says that my version is `0.5.0` even if I explicitly downloaded it from the 0.6.0 release section.

**Comment by MitMaro** [comment]
Seems like the file uploaded for macOS for version 0.6.0 is incorrect.

Thanks for letting me know!

**Comment by MitMaro** [comment]
Since deployments are now automatic, this shouldn't be an issue going forward. I will close this once a new version of the tool is released.

**Comment by MitMaro** [comment]
I've released [version 0.7.0](https://github.com/MitMaro/git-interactive-rebase-tool/releases/tag/0.7.0).
